src/WCSimRunAction.cc

```diff
@@ -155,14 +155,14 @@ void WCSimRunAction::BeginOfRunAction(const G4Run* /*aRun*/)
   flatfile->SetCompressionLevel(2); //default is 1 (minimal compression)
   masterTree = new TTree("MasterTree","Main WCSim Tree");
   if(wcsimdetector->GetIsNuPrism()){
-    if(GetSaveRooTracker())
+    if(GetSaveRooTracker()){
```

Comment:
This if block produces a segmentation fault it the SaveRooTracker option is turned off (e.g., if running particle guns), as `geomTree` doesn't get initialized.